docs/drivers.md

```diff
@@ -0,0 +1,66 @@
+# Driver support
+
+You may add integration with other http caches using the extension points provided by this bundle.
+
+The following extension points are available
+ - PurgeClient
+ - TagHandler
+ - FOS TagHandler
+
+If you write a new PurgeClient driver, you **must** also create a corresponding TagHandler and vice
+versa. Creating a FOS TagHandler is optional.
+
+
+## PurgeClient
+
+The PurgeClient is responsible for sending purge requests to the http cache when content is about to be invalidated.
+The PurgeClient must implement EzSystems\PlatformHttpCacheBundle\PurgeClient\PurgeClientInterface and can be registered
+with the following code in services.yml:
+
+```
+services:
+    ezplatform.http_cache_myhttpcachebundle.purge_client.myhttpcache:
+        class: EzSystems\PlatformMyHttpCacheBundle\PurgeClient\MyHttpCachePurgeClient
+        arguments: ["@ezplatform.http_cache.cache_manager"]
```

Comment:
Nitpick: Let's advocate in doc single quotes as well here. Sorry to be persistent and boring, just trying to clean the mess I made some time ago ;)